test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/meta/ContractResources.java

```diff
@@ -205,6 +209,122 @@
+			"\"inputs\":[{\"indexed\":false,\"name\":\"code\",\"type\":\"bytes32\"}," +
+			"{\"indexed\":false,\"name\":\"timestamp\",\"type\":\"uint256\"}]," +
+			"\"name\":\"JurisdictionRemoved\",\"type\":\"event\"},{\"anonymous\":false,\"inputs\":[" +
+			"{\"indexed\":false,\"name\":\"oldTaxRate\",\"type\":\"uint256\"}," +
+			"{\"indexed\":false,\"name\":\"newTaxRate\",\"type\":\"uint256\"}," +
+			"{\"indexed\":false,\"name\":\"timestamp\",\"type\":\"uint256\"}]," +
+			"\"name\":\"TaxRateChanged\",\"type\":\"event\"},{\"anonymous\":false,\"inputs\":[" +
+			"{\"indexed\":true,\"name\":\"previousOwner\",\"type\":\"address\"}," +
+			"{\"indexed\":true,\"name\":\"newOwner\",\"type\":\"address\"}]," +
+			"\"name\":\"OwnershipTransferred\",\"type\":\"event\"}]";
+	public static final String MINT_CONSTRUCTOR_ABI = "{\"inputs\":[" +
+			"{\"name\":\"_jurisdictions\",\"type\":\"address\"}," +
+			"{\"name\":\"_admin\",\"type\":\"address\"}]," +
+			"\"payable\":false,\"stateMutability\":\"nonpayable\",\"type\":\"constructor\"}";
+	public static final String MINT_ADD_ABI = "{\"constant\":false,\"inputs\":[" +
+			"{\"name\":\"minter\",\"type\":\"address\"}," +
+			"{\"name\":\"name\",\"type\":\"string\"}," +
+			"{\"name\":\"jurisdiction\",\"type\":\"bytes32\"}]," +
+			"\"name\":\"add\",\"outputs\":[],\"payable\":false," +
+			"\"stateMutability\":\"nonpayable\",\"type\":\"function\"}";
+	public static final String MINT_ISVALID_ABI = "{\"constant\":true,\"inputs\":[" +
+			"{\"name\":\"minter\",\"type\":\"address\"}],\"name\":\"isValid\",\"outputs\":[" +
+			"{\"name\":\"\",\"type\":\"bool\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"}";
+	public static final String MINT_SEVEN_ABI = "{\"constant\":true,\"inputs\":[],\"name\":\"seven\"," +
+			"\"outputs\":[{\"name\":\"\",\"type\":\"uint256\"}]," +
+			"\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"}";
+	public static final String MINT_OWNER_ABI ="{\"constant\":true,\"inputs\":[]," +
+			"\"name\":\"owner\",\"outputs\":[{\"name\":\"\",\"type\":\"address\"}]," +
+			"\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"}";
+	public static final String MINT_CONFIGURE_ABI = "{\"constant\":false,\"inputs\":[" +
+			"{\"name\":\"_jurisdictions\",\"type\":\"address\"}]," +
+			"\"name\":\"configureJurisdictionContract\",\"outputs\":[]," +
+			"\"payable\":false,\"stateMutability\":\"nonpayable\",\"type\":\"function\"}";
+	public static final String PAY_TEST_GET_BALANCE = "{\"constant\":true,\"inputs\":[],\"name\":\"getBalance\"," +
+			"\"outputs\":[{\"name\":\"\",\"type\":\"uint256\"}]," +
+			"\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"}";
+	public static final String PAY_TEST_DEPOSIT = "{\"constant\":false,\"inputs\":[" +
+			"{\"name\":\"amount\",\"type\":\"uint256\"}],\"name\":\"deposit\",\"outputs\":[]," +
+			"\"payable\":true,\"stateMutability\":\"payable\",\"type\":\"function\"}";
```

Comment:
I believe these 2 are duplicates of `DEPOSIT_ABI` and `GET_BALANCE_ABI`. Please double check and remove them if identical.
Also, some `nits`:
- Typo `JURIDICTION` -> `JURISDICTION`
- The ABI string format is preferable to split the `input` and `output` into two separate lines, (the `constant` thing is on the line before the input, and the `payable` thing is on the line after the output. If those two lines are too long, split more at the end of each parameter in the input/output.

Reply:
Thanks @QianSwirlds. Fixed them